src/storagetm/storagetm.c

```diff
@@ -1136,6 +1137,12 @@ static int run(int argc, char* argv[]) {
         if (r < 0)
                 return log_error_errno(r, "Failed to install exit signal handlers: %m");
 
+        r = path_is_mount_point("/sys/kernel/config");
+        if (r < 0)
+                return log_error_errno(r, "Failed to check if the configfs filesystem is mounted at /sys/kernel/config: %m");
```

Comment:
nitpick: we usually suffix paths that reference dirs with a `/` in log messages and docs. Please do so here too, i.e. `/sys/kernel/config/`